packages/amplication-data-service-generator/src/admin/entity/entity-show-component/create-entity-show-component.ts

```diff
@@ -140,8 +140,8 @@ function createToManyReferenceField(
   const element = jsxElement` 
   <ReferenceManyField
     reference="${field.properties.relatedEntity.name}"
-    target="${entity.name}Id"
-    label="${field.properties.relatedEntity.pluralDisplayName}"
+    target="${field.properties.relatedField.name}Id"
```

Comment:
we need to double-check if this concatenation of the related field name and "Id" will always work.

Reply:
I will check again

Reply:
The id here are handled at https://github.com/amplication/ra-data-graphql-amplication/blob/5852c1dfe5a95da7e2776fa441ca3243206ca96d/src/buildVariables.ts#L150.

Reply:
id is required elsewhere because field.name represents an object

Reply:
my concern is about the concatenation of "realtedField.name" with "Id" - because if I understand correctly - the related field name should already include the Id suffix. I could be wrong - I just want to make sure we verify that